account_account_constraint_code/models/account_account.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class AccountAccount(models.Model):
+
+    _inherit = "account.account"
+
+    @api.constrains("code")
+    def _disable_code_modification(self):
+        self.env["account.move.line"].flush(["account_id", "journal_id"])
```

Comment:
Why flushing `journal_id` if not used into the query?

Why why doing it by `SQL` instead of using ORM by a `search(...)`?